util/Migrator/DbScripts/2025-06-26_01_CascadeDeleteNotificationStatus.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN
```

Comment:
ℹ️ Farfetched comment: while needed for your local installation, you don't need to rename these to increment upon update. It is highly unlikely to be an issue, but someone could technically come in and add a conflicting `00_` script.